packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -50,6 +51,10 @@ export function addRoutesToApp(app: Express) {
     '/v1/confirmedWeeklyReport/:organisationUnitCode',
     handleWith(ConfirmedCountryWeeklyReportRoute),
   );
+  app.get(
+    '/v1/unconfirmedWeeklyReport/:organisationUnitCode',
+    handleWith(UnconfirmedCountryWeeklyReportRoute),
+  );
```

Comment:
I actually was not aware of the terms we're exposing externally... so that's a good point you brought up. I second your opinion that we should make it consistent

Reply:
I agree with you @rohan-bes , this is consistent with the naming conventions used in indicators and data elements. We can probably skip `unconfirmed` even in the code

Reply:
Agreed :+1: